### PR TITLE
feat: Rewrite `configMixin` to a composable

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR will replace the usage of `configMixin` with the `useConfig` composable.

**Files updated**
1. dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue
2. dashboard/components/widgets/conversation/MessagesView.vue
3. dashboard/routes/dashboard/settings/inbox/Settings.vue **(Not used)**
4. dashboard/routes/dashboard/settings/inbox/FinishSetup.vue **(Not used)**
5. dashboard/routes/dashboard/settings/inbox/settingsPage/CollaboratorsPage.vue
6. dashboard/routes/dashboard/settings/profile/NotificationPreferences.vue **(Not used)**
7. dashboard/routes/dashboard/settings/profile/AudioNotifications.vue **(Not used)**
8. dashboard/routes/dashboard/settings/sla/Index.vue **(Not used)**
9. dashboard/routes/dashboard/settings/account/Index.vue
10. survey/views/Response.vue **(Not used)**

Fixes https://linear.app/chatwoot/issue/CW-3464/rewrite-configmixin-mixin-to-a-composable

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Test in the component related pages

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/composables/useConfig.js
Comment (scmmishra) [inline app/javascript/dashboard/composables/useConfig.js:1]: I don't think we need these to be computed variables. @iamsivin WDYT?

Comment (iamsivin) [reply app/javascript/dashboard/composables/useConfig.js:1]: @scmmishra Sound good. Updated the PR https://github.com/chatwoot/chatwoot/pull/9921/commits/1c9256c45c2c3fdabe1bcda6b56ad45ed3f0751e

File: app/javascript/dashboard/composables/spec/useConfig.spec.js
Comment (scmmishra) [inline app/javascript/dashboard/composables/spec/useConfig.spec.js:1]: @iamsivin we will have to update the tests too

Comment (iamsivin) [reply app/javascript/dashboard/composables/spec/useConfig.spec.js:1]: @scmmishra Sorry missed it. Updated it now.

File: app/javascript/dashboard/composables/spec/useConfig.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { ref } from 'vue';
+import { useConfig } from '../useConfig';
+
+vi.mock('../useConfig', () => ({
+  useConfig: vi.fn(),
+}));
+
+describe('useConfig', () => {
+  beforeEach(() => {
+    useConfig.mockReturnValue({
```
[the remark this answers is not in the document]
Comment (iamsivin) [reply app/javascript/dashboard/composables/spec/useConfig.spec.js]: @scmmishra Updated it.

**Review by scmmishra — APPROVED** [review]

**Pull request merged**